app/models/concerns/user_access_key_overrides.rb

```diff
@@ -7,17 +7,6 @@ module UserAccessKeyOverrides
 
   attr_accessor :user_access_key
 
-  def password_digest(password)
```

Comment:
This was necessary before because we were still calling up to Devise, now we are not, so this method is not required.